conf/Readme.md

```diff
@@ -0,0 +1,44 @@
+# Configurazione di un PSP
+
+La configurazione di un PSP all'interno della piattaforma pagoPA avviene tramite la trasmissione di:
+
+- Il CatalogoDatiInformativi ( CDI) , file xml il cui schema è rappresentato dal file CatalogoDatiInformativi.xsd
+- Report della configurazione, file html dove vengono riassunte le iformazioni essenziali dei servizi configurati
+
+## building
+
+all'interno della directory `conf` eseguire i comandi
+
+`xslproc -o ./out/CatalogoDatiInformativi.html ./xs3p/xs3p.xsl CatalogoDatiInformativi.xsd`
+Per la compilazione della pagina di documentazione
+
+`xslproc -o ./out/form_CatalogoDatiInformativi.html ./xs2html2xml/form_CatalogoDatiInformativi.html`
+per la creazione di una form dinamica.
+
+oppure utilizzare lo script `./build.sh`
```

Comment:
`warning: failed to load external entity "./xsd2html2xml/xsd2html2xml.xsl"
cannot parse ./xsd2html2xml/xsd2html2xml.xsl
warning: failed to load external entity "./xs3p/xs3p.xsl"
cannot parse ./xs3p/xs3p.xsl`